tests/web/test_browse_folder_route.py

```
@@ -0,0 +1,233 @@
+    def test_calls_osascript_posix_path(self, client: TestClient) -> None:
+        """Must call osascript with 'POSIX path of (choose folder)'."""
+        mock_result = self._make_run_result("/tmp/\n")
+        with (
+            patch("file_organizer.api.routers.setup.sys.platform", "darwin"),
+            patch(
+                "file_organizer.api.routers.setup.subprocess.run",
+                return_value=mock_result,
+            ) as mock_run,
+        ):
+            client.get("/api/setup/browse-folder")
+        args, kwargs = mock_run.call_args
```

Ruff (0.15.9)

[warning] Unpacked variable `kwargs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)